LibraBFT/Impl/Consensus/SafetyRules/SafetyRules.agda

```diff
@@ -145,15 +159,15 @@ module constructAndSignVoteM-continue2 (voteProposal : VoteProposal) (validatorS
   step₁ safetyData1 = do
     lSafetyData ∙= safetyData1  -- TODO-1: resolve discussion about pssSafetyData vs lSafetyData
     extensionCheckM voteProposal ∙?∙ (step₂ safetyData1)
-
```

Comment:
You've fairly consistently left no blank lines between steps.  (There is still at least one, between steps 0 and 1 in `constructAndSignVoteM-continue2`.)  I'd actually prefer to consistently *include* the blank lines, as I think it makes it easier to compare to the Haskell, as we need to visually "elide" the lines that introduce the next step and the arguments from the previous one.  If you feel otherwise, that's OK, I'll try to get used to it.

Reply:
I will add those extra lines in.